charters/Proposed-WG-WebIdentityCredentials.md

```diff
@@ -2,7 +2,7 @@
 
 The *W3C Web Identity Credential Working Group* will develop
 recommendation-track specifications defining an API that allows websites
-to request a federated identity credential or assertion with the purpose of authenticating a user and/or requesting a set of claims in a compatible way OIDC or SAML.
+to request a federated identity credential or assertion with the purpose of authenticating a user and/or requesting a set of claims compatible with OIDC and SAML (specifically the [SP initiated Web Browser SSO profile](http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf)).
```

Comment:
```suggestion
to request a federated identity credential or assertion with the purpose
of authenticating a user and/or requesting a set of claims compatible with
OIDC and SAML, specifically the
[SP initiated Web Browser SSO profile](http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf).
```

Reply:
There are multiple OIDC profiles: not sure which ones should be considered in scope. The relevant SAML profile is clear.

Reply:
consensus at TPAC that the "with/and" change is fine, but adding protocol specifics (the parenthetic text) should be avoided.

Reply:
Acting on @timcappalli's note
```suggestion
to request a federated identity credential or assertion with the purpose
of authenticating a user and/or requesting a set of claims compatible with
OIDC and SAML.
```

Reply:
Forgive my GIT clumsiness  -- has committing Tall Ted's suggestion done the job?